qiskit/providers/aer/noise/errors/quantum_error.py

```diff
@@ -361,43 +361,43 @@ def power(self, n):
         return ret
 
     def tensor(self, other):
-        """Return the tensor product quantum error channel self ⊗ other.
+        """Return the tensor product quantum error channel self ? other.
```

Comment:
Looks like something odd is going on with the math operator font characters here